content/bizops/user_definitions.md

```diff
@@ -1,54 +1,60 @@
 # User metrics definitions
 
-We use metrics to guide prioritization and planning. By defining metrics against the kinds of behaviors that we want to drive, we can plan features and measure results against the outcomes we're looking for. The three most important metrics we track are acquisition (of visitors), activation (visitor experience aha! moment), and retention (user remains active).
+We use metrics to guide prioritization and planning. By defining metrics against the kinds of behaviors that we want to drive, we can plan features and measure results against the outcomes we're looking for. The three most important metrics we track are acquisition (of visitors), activation (visitor experiences habit creation), and retention (user remains active).
 
 ## User states
 
-People using Sourcegraph can be segmented into a number of different states, the following of which are relevant over a measured time period (for example, monthly or weekly):
+People using Sourcegraph can be segmented into a number of different states, the following of which are relevant over a measured time period (for example, monthly or weekly). They are not mutually exclusive in some cases - a user can be both registered, activated and churned at the same time.
```

Comment:
Maybe add a column indicating which ones can be are mutually exclusive and which are not? It's one of the most confusing things that this is unclear/inconsistent.

Reply:
Good callout. Will come up with a proposed solution

Reply:
Added this section in the latest commit. Let me know your thoughts!